### ISSUE TITLE
suggestion : ouverture dans un nouvel onglet

### ISSUE THREAD

**Issue by dominique-r** [lovasoa/ophirofox#108]
Bonjour,

Une suggestion d'amélioration (de mon point de vue) :
actuellement (sur Mozilla) le lien "Lire sur Europresse" ouvre l'interface d'Europresse en lieu et place de la page de consultation.
Il me semble que l'ouverture dans un nouvel "onglet" (ou page) serait plus ergonomique pour l'utilisateur : il ne perdrait pas sa page d'origine.
Donc "en gros" mettre un target="_blank" sur le lien "Lire sur Europresse".

Au début je pensais même plutôt a un target nommé, pour que des clics sur plusieurs liens "Lire sur Europresse" s'ouvrent sur le même onglet (ou page). Mais à la réflexion le dernier clic "effacerait" le résultat du précédent. 

D'où une seconde suggestion : est-ce que, 
tout comme l'extension lance la recherche dans l'interface europresse, 
elle ne pourrait pas automatiquement cocher la (les ?) réponse(s) et épingler celle(s)-ci ?

Ainsi on devrait pouvoir ouvrir tous les liens  "Lire sur Europresse" sur le même onglet (ou page) sans perte des résultats précédents ?

Huummm  ... chaque lien "Lire sur Europresse" semble ouvrir une nouvelle session sur la plateforme ... 

Cordialement

**Comment by lovasoa** [comment]
pour ouvrir dans un nouvel onglet, vous pouvez faire un clic droit, et "ouvrir dans un nouvel onglet", ou juste cliquer avec le bouton central de votre souris.

**Comment by lovasoa** [comment]
https://medium.com/cantileverco/the-new-tab-conundrum-12487947feda

**Comment by dominique-r** [comment]
Pour le clic droit : oui c'est vrai.

Pour "The “New Tab” Conundrum" et 
"Allowing the browser to work as it naturally does
When you force a link to open in a new tab, the user can no longer use the browser’s native “back” functionality to return to your site. They must find the original tab and click that instead."

... je suis plus dubitatif.

**Comment by lovasoa** [comment]
La raison pour laquelle on fait juste un lien normal, c'est que c'est facile de l'ouvrir dans un nouvel onglet si on veut,  ou dans le même onglet si on veut. L'inverse ne fonctionne pas.

**Comment by dominique-r** [comment]
Okay

**Comment by Kalytis** [comment]
Dommage, je trouve aussi que ça serait très pratique d'avoir un nouvel onglet par défaut, pour lire les articles sur Europresse.

**Comment by lovasoa** [comment]
Cliquez sur le lien *Lire sur Europresse* avec le bouton central de la souris. Le lien s'ouvrira dans un nouvel onglet. Et la bonne nouvelle, c'est que ça marche avec tous les liens :)

**Comment by Kalytis** [comment]
C'est un contournement, et c'est déjà ce que l'on fait, mais ça pourrait être pratique d'avoir la fonctionnalité par défaut.

**Comment by lovasoa** [comment]
Si ça s'ouvrait dans un nouvel onglet par défaut, il n'y aurait pas de "contournement" simple pour ceux qui le veulent dans le même onglet. C'est pour ça que je préfère le garder comme ça.

**Comment by Kalytis** [comment]
Très bonne raison, je ne l'avais pas vu comme ça. Merci !

**Comment by Deuchnord** [comment]
Hello, je serais aussi intéressé par la possibilité d'ouvrir dans un nouvel onglet.
Ne pourrait-on pas ajouter une option (désactivée par défaut pour les raisons d'accessibilité évoquées plus haut) dans les préférences de l'extension dans ce sens ?

**Comment by lovasoa** [comment]
Oui, je n'y verrais pas de problème ! Si quelqu'un veut faire une pull request !

**Comment by Deuchnord** [comment]
Je vais tenter de faire une contrib :)

**Comment by Write** [comment]
Bonjour à tous,

J'ai implémenté la fonctionnalité dans le  #321.
